astroquery/nasa_exoplanet_archive/core.py

```diff
@@ -59,10 +72,40 @@
     "pi_rel": None,
     "ppm": cds.ppm,
     "seconds": u.s,
+    "Solar mass": u.M_sun,
     "solarradius": u.R_sun,
+    "Solar Radius": u.R_sun,
+    "log10(cm/s**2)": u.dex(u.cm / u.s ** 2),
+    "dex": u.dex(None),
+    "sexagesimal": None
 }
+
 CONVERTERS = dict(koi_quarters=[ascii.convert_numpy(str)])
-OBJECT_TABLES = {"exoplanets": "pl_", "compositepars": "fpl_", "exomultpars": "mpl_"}
+
+# 'ps' and 'pscomppars' are the main tables of detected exoplanets.
+# Calls to the old tables ('exoplanets', 'compositepars', 'exomultpars') will
+# return errors and urge the user to call the 'ps' or 'pscomppars' tables
+OBJECT_TABLES = {"ps": "pl_", "pscomppars": "pl_", "exoplanets": "pl_",
+                 "compositepars": "fpl_", "exomultpars": "mpl_"}
+MAP_TABLEWARNINGS = {"exoplanets": "Planetary Systems (PS)",
```

Comment:
I think this is a bit confusing in the warning message, but can be adjusted if there are any user complaints.